### PR TITLE
4: Deactivate clients.

### PR DESCRIPTION
**Pull request by robherba**
**Steps to test**
- Create a new client node.
- Check if the new VBO action is executed successfully.

**DoD**
- In clients view (created in a previous ticket) ensure we have a VBO action to enable/disable clients.

[issue](https://github.com/ManatiCR/manatime/issues/4)

### CODE REVIEW

**Comment by kporras07** [comment]
You forgot to recreate the view.

File: modules/features/manatime_feature_clients/manatime_feature_clients.module [hunk 1]
```diff
@@ -17,3 +17,41 @@ function manatime_feature_clients_permission() {
...
 }
+
+/**
+ * Implements hook_action_info().
+ */
+function manatime_feature_clients_action_info() {
+  return array(
+    'manatime_feature_clients_action_enable' => array(
+      'type' => 'entity',
+      'label' => t('Enable clients'),
```
Comment (kporras07) [inline modules/features/manatime_feature_clients/manatime_feature_clients.module]: Activate Clients

File: modules/features/manatime_feature_clients/manatime_feature_clients.module [hunk 2]
```diff
@@ -17,3 +17,41 @@ function manatime_feature_clients_permission() {
...
+      'behavior' => array('changes_property'),
+      'configurable' => FALSE,
+      'vbo_configurable' => FALSE,
+      'triggers' => array('any'),
+    ),
+    'manatime_feature_clients_action_disable' => array(
+      'type' => 'entity',
+      'label' => t('Disable clients'),
```
Comment (kporras07) [inline modules/features/manatime_feature_clients/manatime_feature_clients.module]: Deactivate Clients

**Comment by robherba** [comment]
Changes has been applied. @kporras07

**Comment by kporras07** [comment]
:shipit:

**Pull request merged**